### PR TITLE
Started work on emoji reactions to render

### PR DESCRIPTION
**Pull request by aaryanporwal**
Co-authored-by: Sam Poder <sam@hackclub.com>

### CODE REVIEW

File: api/src/services/updates/updates.ts [hunk 1]
```diff
@@ -44,27 +47,28 @@ export const createUpdate: MutationResolvers['createUpdate'] = async ({
...
+    if (getDayOfDate(update.postTime) - currentDay == -1) {
       streak += 1
       currentDay -= 1
     }
-  }) 
-  console.log(streak) 
+  })
+  console.log(streak)
   return result[0]
 }
```
Comment (aaryanporwal) [inline api/src/services/updates/updates.ts:73]: prettier did this

File: web/src/components/Update/Updates/Updates.tsx [hunk 2]
```diff
@@ -177,37 +177,41 @@ const UpdatesList = ({ updates }: FindUpdates) => {
                     )
                   }
                 >
-                  {reaction.emojiTypeName}
+                  {JSON.stringify(reaction)}
```
Comment (sampoder) [inline web/src/components/Update/Updates/Updates.tsx:180]:
```suggestion
                  {reaction.emojiTypeName}
```

File: web/src/components/Update/Updates/Updates.tsx [hunk 3]
```diff
@@ -181,33 +181,37 @@ const UpdatesList = ({ updates }: FindUpdates) => {
...
-                onClick={() => onAddEmojiClick(update.id, currentUser?.id)}
-              >
-                Add Emoji
-              </button>
-            </nav>
-          )}
+          <nav className="rw-table-actions mt-2 mb-1 justify-center">
+            {update.accountsID == currentUser?.id && (
+              <>
+                {' '}
```
Comment (sampoder) [inline web/src/components/Update/Updates/Updates.tsx:187]:
```suggestion
```

File: web/src/components/Update/Updates/Updates.tsx [hunk 4]
```diff
@@ -181,33 +181,36 @@ const UpdatesList = ({ updates }: FindUpdates) => {
...
+                <button
+                  type="button"
+                  title={'Delete update ' + update.id}
+                  className="rw-button rw-button-small rw-button-red"
+                  onClick={() => onDeleteClick(update.id)}
+                >
+                  Delete
+                </button>
+              </>
+
```
Comment (sampoder) [inline web/src/components/Update/Updates/Updates.tsx:203]:
```suggestion
```

File: web/src/components/Update/Updates/Updates.tsx [hunk 5]
```diff
@@ -181,33 +181,35 @@ const UpdatesList = ({ updates }: FindUpdates) => {
...
+              </>
+            )}
+                <button
+                  type="button"
+                  title={'Add emoji to update ' + update.id}
+                  className="rw-button rw-button-small text-purple"
+                  onClick={() => onAddEmojiClick(update.id, currentUser?.id)}
+                >
+                  Add Emoji
+                </button>
```
Comment (sampoder) [inline web/src/components/Update/Updates/Updates.tsx:211]:
```suggestion
            <button
              type="button"
              title={'Add emoji to update ' + update.id}
              className="rw-button rw-button-small text-purple"
              onClick={() => onAddEmojiClick(update.id, currentUser?.id)}
            >
              Add Emoji
            </button>
```

File: api/src/services/updates/updates.ts (hunk 1 above)
Comment (sampoder) [reply api/src/services/updates/updates.ts:73]: 👍

**Review by sampoder — APPROVED** [review]
Thanks @aaryanporwal! I'll add onto this & get in touch.

**Pull request merged**